pycfmodel/model/types.py

```diff
@@ -1,56 +1,80 @@
+import binascii
+from base64 import b64decode
 from datetime import date, datetime
-from ipaddress import IPv4Network, IPv6Network, _BaseNetwork
-from typing import Any, Dict, Generator, List, TypeVar, Union
+from ipaddress import IPv4Network, IPv6Network
+from typing import Any, List, TypeVar, Union
 
-from pydantic.networks import NetworkType
-from pydantic.typing import AnyCallable
+from pydantic import BeforeValidator, Field, GetCoreSchemaHandler, Strict
+from pydantic._internal import _schema_generation_shared
+from pydantic.json_schema import JsonSchemaValue
+from pydantic_core import core_schema
+from typing_extensions import Annotated
 
 from pycfmodel.model.base import FunctionDict
 
 
-class LooseIPv4Network(_BaseNetwork):
+class LooseIPv4Network:
+    __slots__ = ()
```

Comment:
why do we want slots here?

Reply:
It's to make it more efficient, we don't want to store anything in memory